source/base/data_out_base.cc

```diff
@@ -850,11 +850,6 @@ namespace
      * technique.
      */
     std::ostream &stream;
-
-    /**
-     * The flags controlling the output
-     */
-    const DataOutBase::GmvFlags flags;
   };
```

Comment:
I think it would be useful to retain this field, to make sure that if one ever added an element to it, it would in fact be stored somewhere by the constructor of this class. Or do you get a warning for this as well?

The other changes in this patch seem quite reasonable.